packages/react-components/src/organisms/UserProfileRecentWorks.tsx

```diff
@@ -129,26 +136,58 @@ const UserProfileRecentWork: React.FC<UserProfileRecentWorkProps> = ({
 
 const UserProfileRecentWorks: React.FC<UserProfileRecentWorksProps> = ({
   orcidWorks = [],
-}) => (
-  <Card>
-    <div css={headerStyles}>
-      <Headline2 styleAsHeading={3}>
-        Recent Publications ({orcidWorks.length})
-      </Headline2>
-      <Paragraph accent="lead">Via ORCID</Paragraph>
-      <span css={{ display: 'grid', svg: { fill: lead.rgb } }}>
-        {orcidIcon}
-      </span>
-    </div>
-    <ul css={listStyles}>
-      {orcidWorks
-        .flatMap((work, idx) => [
-          <Divider key={`sep-${idx}`} />,
-          <UserProfileRecentWork key={`wrk-${idx}`} {...work} />,
-        ])
-        .slice(1)}
-    </ul>
-  </Card>
-);
+  orcid,
+}) => {
+  const { isOwnProfile } = useContext(UserProfileContext);
+
+  return orcidWorks.length || isOwnProfile ? (
```

Comment:
Maybe make the `return null` an early return. Feel like the nested ternary is getting confusing